cirq/google/api/v2/program.proto

```diff
@@ -24,6 +24,14 @@ message Program {
   }
 }
 
+// A Batch of multiple circuits that should be run as one program
+message BatchedProgram {
```

Comment:
Nit: call this `BatchProgram` instead? Similarly for `BatchRunContext` and `BatchResult`.

Reply:
Done.